### ISSUE TITLE
Improve the CKEditor 5 demo to include accessibility features by default

### ISSUE THREAD

**Issue by CharlesBelov** [ckeditor/ckeditor5#15294]
## 📝 Provide a description of the improvement

The [CKEditor 5 demo](https://ckeditor.com/ckeditor-5/demo/html-support/) includes a large number of formatting features as part of the WYSIWYG. However, it is missing at least a couple accessibility features. It would be better if it also included the following:

- ability to insert language mark up from the toolbar
- the image insert would include the alternative text field

## 📃 Other details

* Browser: Firefox 119.0
* OS: macOS 14.1
* CKEditor version: Whatever is installed on the demo
* Installed CKEditor plugins: Whatever is installed on the demo

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.

**Comment by mgifford** [comment]
This would be quite useful.

**Comment by CKEditorBot** [comment]
There has been no activity on this issue for the past year. We've marked it as stale and will close it in 30 days. We understand it may still be relevant, so if you're interested in the solution, leave a comment or reaction under this issue.

**Comment by CharlesBelov** [comment]
It appears the Alt text issue has been resolved.
The language attribute is still not directly supported without inserting HTML code.

So still an issue.

**Comment by CKEditorBot** [comment]
There has been no activity on this issue for the past year. We've marked it as stale and will close it in 30 days. We understand it may still be relevant, so if you're interested in the solution, leave a comment or reaction under this issue.

**Comment by CKEditorBot** [comment]
We've closed your issue due to inactivity. We understand that the issue may still be relevant. If so, feel free to open a new one (and link this issue to it).
